python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -60,7 +61,7 @@ def __str__(self):
         return (
             f"InternalItemState["
             f"initial_value={self.initial_value}, "
-            f"recent_average={self.recent_average}, "
+            f"recent_average={self.recent_median}, "
```

Comment:
Please update the text "recent_average". Consider a global search for "average"

Reply:
Very sloppy of me. Thanks for catching that.